### PR TITLE
dev-python/configobj: Revbump to fix arm tests

### PR DESCRIPTION
**Pull request by ajakk**
Switch SRC_URI to Github release tarball, add patch to fix tests on arm,
EAPI bump. Add DISTUTILS_USE_SETUPTOOLS=no like the eclass suggests.

Bug: https://bugs.gentoo.org/732092
Package-Manager: Portage-3.0.1, Repoman-2.3.23
Signed-off-by: John Helmert III <jchelmert3@posteo.net>

### CODE REVIEW

File: dev-python/configobj/configobj-5.0.6-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{6,7,8} )
```
Comment (mgorny) [inline dev-python/configobj/configobj-5.0.6-r1.ebuild:6]: Add 3.9 while at it?

File: dev-python/configobj/configobj-5.0.6-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+DISTUTILS_USE_SETUPTOOLS=no
+
+inherit distutils-r1
+
+DESCRIPTION="Simple config file reader and writer"
+HOMEPAGE="http://www.voidspace.org.uk/python/configobj.html https://pypi.org/project/configobj/"
+SRC_URI="https://github.com/DiffSK/${PN}/archive/v${PV}.tar.gz -> ${P}-gh.tar.gz"
```
Comment (mgorny) [inline dev-python/configobj/configobj-5.0.6-r1.ebuild:13]: `.gh`, please. `-gh` suggests that the directory inside is `-gh`.

File: dev-python/configobj/configobj-5.0.6-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
+
+RDEPEND="dev-python/six[${PYTHON_USEDEP}]"
+
+PATCHES=( "${FILESDIR}/${P}-fix-py2-tests.patch" )
+
+python_prepare_all() {
+	# Not to install un-needed _version.py
```
Comment (mgorny) [inline dev-python/configobj/configobj-5.0.6-r1.ebuild:24]: I wonder why this is claimed necessary…

Comment (ajakk) [reply dev-python/configobj/configobj-5.0.6-r1.ebuild:24]: Tests pass without this whole function. Safe to drop it?

**Pull request closed without merging**